### ISSUE TITLE
[Feature][Question] Workers utilisation visualisation

### ISSUE THREAD

**Issue by dimkin-eu** [microsoft/playwright#14706]
In Allure report, there is a timeline panel, which shows how multiple tests are run in parallel
![image](https://user-images.githubusercontent.com/7899764/172623614-87f0c66a-54fd-4171-a079-36d9b8595185.png)

Is there a way to show in the more or less same way workers utilization?

This could help in tests optimising, to check if some of them are taking too long
( also shoving pass/fail/retry there is a nice feature )

**Comment by pavelfeldman** [comment]
Did you enable `fullyParallel` in your config? That would make all these bars green all the time.

**Comment by dimkin-eu** [comment]
> Did you enable fullyParallel in your config? 

nope ( also this is not possible for all specs )

Allure's picture is some random from the internet for reference/idea :)

**Comment by baev** [comment]
related Allure issue https://github.com/allure-framework/allure-js/issues/479

**Comment by vovsemenv** [comment]
@dimkin-eu after `allure-playwright@2.0.0-beta.19` reporter generate timeline for playwright in parallel and shared mode 
![image](https://user-images.githubusercontent.com/20096510/183687661-d5255afb-c907-42ab-9694-9b07ba1d61f4.png)
![image](https://user-images.githubusercontent.com/20096510/183687700-16245cbd-4ac6-4cb1-a78e-b52ec939df61.png)

@pavelfeldman can close ticket

**Comment by dimkin-eu** [comment]
initially i needed this for pure playwright, as not using allure per se. playwright's html reporter is very cool

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your involvement. This issue was closed due to limited engagement (upvotes/activity), lack of recent activity, and insufficient actionability. To maintain a manageable database, we prioritize issues based on these factors.

If you disagree with this closure, please open a new issue and reference this one. More support or clarity on its necessity may prompt a review. Your understanding and cooperation are appreciated.
